backend/database.js

```diff
@@ -92,6 +84,47 @@ export default (sbp('sbp/selectors/register', {
       }
     })
   },
+  'backend/db/streamEntriesBetween': async function (startHash: string, endHash: string, offset: number): Promise<*> {
+    offset = parseInt(offset) === isNaN ? 0 : Math.max(0, parseInt(offset))
```

Comment:
This comparison is incorrect, isNaN is a function, and you should use typeof instead

Reply:
Correct. Poor concentration now.